packages/features/vault/src/side-drawers/manage-vault/manage-vault.tsx

```diff
@@ -0,0 +1,109 @@
+import { Box, styled, useTheme } from '@mui/material';
+import {
+  PORTFOLIO_ACTIONS,
+  VAULT_ACTIONS,
+} from '@notional-finance/shared-config';
+import {
+  H4,
+  LabelValue,
+  LargeInputTextEmphasized,
+  SideDrawerButton,
+} from '@notional-finance/mui';
+import { Link, useParams } from 'react-router-dom';
+import { FormattedMessage } from 'react-intl';
+import { useManageVault } from './use-manage-vault';
+import { useVault } from '@notional-finance/notionable-hooks';
+import { messages } from '../messages';
+
+interface VaultParams {
+  vaultAddress?: string;
+  sideDrawerKey?: VAULT_ACTIONS;
+}
+
+export const ManageVault = () => {
+  const theme = useTheme();
+  const { vaultAddress } = useParams<VaultParams>();
+  const { reduceLeverageOptions, manageVaultOptions } = useManageVault();
+  const { vaultName } = useVault(vaultAddress);
+
+  return (
+    <Box>
+      <LargeInputTextEmphasized
+        gutter="default"
+        sx={{ marginBottom: theme.spacing(5) }}
+      >
+        <FormattedMessage
+          {...messages[PORTFOLIO_ACTIONS.MANAGE_VAULT].heading}
+          values={{
+            vaultName,
+          }}
+        />
+      </LargeInputTextEmphasized>
+      <Title>
+        <FormattedMessage defaultMessage={'Reduce leverage'} />
+      </Title>
+      {reduceLeverageOptions.map(({ label, link }, index) => (
+        <SideDrawerButton key={index} sx={{ padding: '0px' }}>
+          <Link
+            to={link}
+            style={{
+              display: 'flex',
+              alignItems: 'center',
+              width: '100%',
+              padding: theme.spacing(2.5),
+            }}
+          >
+            <H4
```

Comment:
The typography object (H4 here) takes a to= prop and will wrap the text in react-router-dom Link component. I'd use that instead here to reduce the amount of code and you can combine the sx props

Reply:
I was planning on coming back to fix these up later but I'll take care of them now.